src/tdastro/astro_utils/opsim.py

```diff
@@ -403,6 +446,58 @@ def bandflux_error_point_source(self, bandflux, index):
         )
 
 
+def create_random_opsim(num_obs, seed=None):
```

Comment:
This is very close to what we have at https://github.com/lincc-frameworks/tdastro/tree/main/tests/tdastro/testing_utils. Should we move things around so all relevant functions are in one place? Either here or in that directory.

Reply:
We should eventually combine them. Let's discuss which of the locations makes the most sense and do that in a later PR.